doc/content/source/materials/ConcreteThermalMoisture.md

```diff
@@ -1,14 +1,495 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # ConcreteThermalMoisture
 
-!alert construction title=Undocumented Class
-The ConcreteThermalMoisture has not been documented. The content contained on this page includes the
-typical automatic documentation associated with a MooseObject; however, what is contained is
-ultimately determined by what is necessary to make the documentation clear for users.
-
 !syntax description /Materials/ConcreteThermalMoisture
 
+## Description
+
+This class computes the set of material coefficients and parameters needed for moisture diffusion and heat transfer in concrete. The equivalent moisture diffusion/heat transfer model [!cite](bazant1979pore,bazant1982finite,xi_moisture_1994_a,xi_moisture_1994_b) is implemented using a set of kernels that provide the individual terms in the system of partial differential equations. This class provides with a full set of constitutive models compiled by [!cite](saouma_structural_2014). The following sections provide detailed descriptions of the governing equations and associated constitutive laws for the coupled moisture diffusion and heat transfer model.
+
+## Heat transfer model
+
+### Governing equation
+
```

Comment:
I am not sure if this material is the right place to provide all the governing equations. It might be better to have an index file for the governing equations.